src/main/java/mate/academy/spring/service/impl/RoleServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.spring.service.impl;
+
+import mate.academy.spring.dao.RoleDao;
+import mate.academy.spring.model.Role;
+import mate.academy.spring.service.RoleService;
+import org.springframework.stereotype.Service;
+
+@Service
+public class RoleServiceImpl implements RoleService {
+    private final RoleDao roleDao;
+
+    public RoleServiceImpl(RoleDao roleDao) {
+        this.roleDao = roleDao;
+    }
+
+    @Override
+    public Role add(Role role) {
+        return roleDao.add(role);
+    }
+
+    @Override
+    public Role getByName(Role.Type roleName) {
+        return roleDao.getByName(roleName).orElseThrow(() ->
+                new RuntimeException("Can't find role by this name" + roleName.name()));
```

Comment:
better to use DataProcessingException
```suggestion
                new DataProcessingException("Can't find role by this name" + roleName.name()));
```